### PR TITLE
CIV-10979 Adding the new Test for the purpose of testing the latest update tab...

### PR DESCRIPTION
**Pull request by pats-john**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/functionalTests/codecept.conf.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const testConfig = require('../config.js');
 //const testHeadlessBrowser = process.env.TEST_HEADLESS ? process.env.TEST_HEADLESS === 'true' : true;
 
 exports.config = {
-  tests: '../functionalTests/tests/*_tests.js',
+  tests: '../functionalTests/tests/cp_latest_update_orders_small_claims_tests.js',
```
Comment (rajakm) [inline src/test/functionalTests/codecept.conf.js:6]: make sure we revert this change before merge

**Review by rajakm — APPROVED** [review]

**Review by dharmendrak — DISMISSED** [review]

**Review by m-meulendijks-v1 — APPROVED** [review]

**Review by rajakm — APPROVED** [review]

**Review by maciejmodzelewski — APPROVED** [review]

**Review by dharmendrak — APPROVED** [review]
